Plugins/GRpluginMaps.txt

```diff
@@ -3826,3 +3827,971 @@ LINE:N060.11.28.595:E011.07.00.094:N060.11.27.792:E011.06.59.728
+LINE:N060.11.07.316:E011.04.24.387:N060.11.08.171:E011.04.28.287
+
+MAP:RWY 01R/19L
+AIRPORT:ENGM
+FOLDER:RWY CLOSURE
+COLOR:CLOSUREAREA
+COORDTYPE:RWYCLOSED:NONE
+COORD:N060.12.04.474:E011.07.23.251
+COORD:N060.12.05.081:E011.07.18.076
+COORD:N060.10.32.935:E011.06.26.175
+COORD:N060.10.32.396:E011.06.30.065
+STYLE:SOLID:2
+LINE:N060.12.00.355:E011.07.17.161:N060.12.01.589:E011.07.20.978
+LINE:N060.12.01.938:E011.07.18.193:N060.11.59.979:E011.07.19.852
+LINE:N060.11.47.100:E011.07.09.783:N060.11.47.980:E011.07.13.062
+LINE:N060.11.48.377:E011.07.10.383:N060.11.46.703:E011.07.12.161
+LINE:N060.11.35.149:E011.07.02.605:N060.11.36.104:E011.07.06.135
+LINE:N060.11.36.459:E011.07.03.481:N060.11.34.773:E011.07.05.209
+LINE:N060.11.18.359:E011.06.52.986:N060.11.19.410:E011.06.56.639
+LINE:N060.11.19.809:E011.06.53.787:N060.11.17.974:E011.06.55.790
+LINE:N060.11.01.905:E011.06.43.445:N060.11.02.832:E011.06.47.082
+LINE:N060.11.03.224:E011.06.44.182:N060.11.01.472:E011.06.46.233
+LINE:N060.10.47.203:E011.06.34.765:N060.10.48.016:E011.06.38.508
+LINE:N060.10.48.430:E011.06.35.452:N060.10.46.776:E011.06.37.739
+LINE:N060.10.31.902:E011.06.25.807:N060.10.32.720:E011.06.29.668
+LINE:N060.10.33.105:E011.06.26.448:N060.10.31.497:E011.06.28.915
+
+//                                                                          
+// ENGM - Construction work at Gardermoen, the summer of 2025
+//
+
+MAP:REMOTE APRON
+AIRPORT:ENGM
+FOLDER:AREA CLOSURE
+COLOR:CLOSUREAREA
+ACTIVE:250505:250612:0:0000:0000
+BLOCKS:ENGM:181,182,183,184,185,186,187,188,189
+COORDTYPE:APW:POLYGON
+COORD:N060.11.28.650:E011.06.36.744
+COORD:N060.11.25.480:E011.06.34.963
+COORD:N060.11.29.050:E011.06.09.462
+COORD:N060.11.32.109:E011.06.11.116
+
+MAP:NORTH PIER WEST
+AIRPORT:ENGM
+FOLDER:AREA CLOSURE
+COLOR:CLOSUREAREA
+ACTIVE:250801:250930:0:0000:0000
+BLOCKS:ENGM:60,64,68,72,76,80,96
+COORDTYPE:APW:POLYGON
+COORD:N060.11.57.138:E011.06.00.801
+COORD:N060.11.56.324:E011.06.06.669
+COORD:N060.11.42.704:E011.05.58.530
+COORD:N060.11.43.508:E011.05.52.737
+
+MAP:NORTH PIER EAST
+AIRPORT:ENGM
+FOLDER:AREA CLOSURE
+COLOR:CLOSUREAREA
+ACTIVE:250801:250930:0:0000:0000
+BLOCKS:ENGM:61,65,69,73,77,79,81,85,87,89,93,95,96
+COORDTYPE:APW:POLYGON
+COORD:N060.11.55.978:E011.06.09.374
+COORD:N060.11.54.942:E011.06.16.837
+COORD:N060.11.41.705:E011.06.09.076
+COORD:N060.11.42.784:E011.06.01.767
+
+MAP:STANDS 11 & 49
+AIRPORT:ENGM
+FOLDER:AREA CLOSURE
+COLOR:CLOSUREAREA
+ACTIVE:250820:251220:0:0000:0000
+BLOCKS:ENGM:11,49
+COORDTYPE:APW:POLYGON
+COORD:N060.11.43.151:E011.05.30.896
+COORD:N060.11.42.899:E011.05.32.806
+COORD:N060.11.41.271:E011.05.31.891
+COORD:N060.11.41.544:E011.05.30.019
+COORDTYPE:APW:POLYGON
+COORD:N060.11.35.549:E011.06.23.031
+COORD:N060.11.34.850:E011.06.28.007
+COORD:N060.11.32.814:E011.06.26.912
+COORD:N060.11.33.521:E011.06.21.875
+
+MAP:STANDS 18-24
+AIRPORT:ENGM
+FOLDER:AREA CLOSURE
+COLOR:CLOSUREAREA
+ACTIVE:250609:250630:0:0000:0000
+BLOCKS:ENGM:18,20,22,24
```

Comment:
"Stands consecutively closed."

but since its only 4 stands, I am good with just closing all 4 the whole period too.